plugins/foreman_discovery/2.0/index.md

```diff
@@ -0,0 +1,306 @@
+---
+layout: plugin
+title: Foreman Discovery 2.0 Manual
+version: 2.0
+---
+
+# 1. {{ page.title }}
+
+This plugin provides automatic bare-metal discovery which is implemented as a
```

Comment:
I'd suggest rewriting this a little, get rid of information like CentOS live image and running from memory from the first sentence - that's not what Discovery is about.  Perhaps focus on saying how it changes the provisioning process and then, if you want to mention the OS technology underneath, do so alongside the lower paragraph about communication methods.